brat/Brat/Error.hs

```diff
@@ -191,9 +191,17 @@ addSrcContext _ _ (Right r) = Right r
 addSrcContext fname cts (Left err@Err{fc=fc}) = Left (SrcErr msg err)
  where
   msg = case fc of
-    Just fc -> unlines (errHeader (fname ++ '@':show fc):showFileContext cts fc)
+    Just fc -> unlines (errHeader (fname ++ prettyFC fc)
+                       :showFileContext cts fc
+                       )
     Nothing -> errHeader fname
```

Comment:
(This is one place that's lost a trailing `:` but not the only one, it's also gone from `addSrcName`, hence the suggestion to reinstate it in `errHeader`)